src/sarif-converter-21.ts

```diff
@@ -70,11 +65,15 @@ export class SarifConverter21 {
             invocations: this.invocationConverter(
                 getEnvironmentDataFromResults(results),
             ),
-            files: files,
+            artifacts: [
+                this.getArtifactProperties(
+                    getEnvironmentDataFromResults(results),
```

Comment:
during refactor, could consider saving in variable since this is used multiple places